lib/pci.py

```diff
@@ -428,6 +428,69 @@ def get_driver(pci_address):
     return ""
 
 
+def ioa_details():
+    """
+    Gets the IPR IOA details and returns
+
+    return: list of dics, with keys ioa, serial, remote serial, pci, status
+    """
+    cmd = "iprconfig -c show-ioas"
+    show_ioas = commands.getoutput(cmd)
```

Comment:
lets use `helper.runcmd()` so that future changes are abstracted...

Reply:
the pci library uses `commands` module, and since we are not sure about the longevity of this library here, we go for this approach.